<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lbTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
color: #252424;
/*background-color:qlineargradient(spread:reflect, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(224, 224, 224, 255), stop:0.495413 rgba(248, 248, 248, 255));*/
background-color:rgb(244, 244, 244);

padding:4px;
font: bold 14px;
font-family: Ubuntu, Helvetica, sans-serif;



border-top-left-radius: 3px solid rgb(244, 244, 244);
border-bottom-left-radius: 3px solid rgb(244, 244, 244) ;

border-top: 1px solid #BFBFBF;
border-bottom:1px solid #BFBFBF;
border-left:1px solid #BFBFBF;

}</string>
       </property>
       <property name="text">
        <string>&lt;Lo que quiero buscar&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbItemsCount">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
color: rgb(147, 147, 147);
background-color:rgb(244, 244, 244);

padding:3px;
font: 10px;
font-family: Ubuntu, Helvetica, sans-serif;

/*
border-top-left: 1px solid rgb(244, 244, 244);
border-top-right: 1px solid rgb(244, 244, 244);
border-bottom-right: 1px solid rgb(244, 244, 244);
border-bottom-left: 1px solid rgb(244, 244, 244);*/
border-top: 1px solid #BFBFBF;
border-bottom: 1px solid #BFBFBF;
}</string>
       </property>
       <property name="text">
        <string>0 items(s) listado(s)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btSalir">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: #252424;
/*background-color:qlineargradient(spread:reflect, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(224, 224, 224, 255), stop:0.495413 rgba(248, 248, 248, 255));*/
background-color:rgb(244, 244, 244);

padding: 4px;
font: bold 14px;
font-family: Ubuntu, Helvetica, sans-serif;
border: 1px solid #BFBFBF;


border-top-left-radius: 0px solid rgb(244, 244, 244);
border-top-right-radius: 3px;
border-bottom-right-radius: 3px;
border-bottom-left-radius: 0px solid rgb(244, 244, 244);

border-left: 0px solid black;
}

QPushButton:hover{
background-color:rgb(254, 254, 254);
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/newPrefix/close.png</normaloff>:/newPrefix/close.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btAceptar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>74</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu,Helvetica,sans-serif</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">QLabel{
color:rgb(85, 85, 85);
font:bold;
font-family: Ubuntu, Helvetica, sans-serif;
}</string>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLineEdit" name="leSearch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit { 
border: 1px solid #999999; 
padding: 2px 2px 2px 2px;
min-width: 90px;
height: 20px;

border-top-left-radius: 3px ;
border-top-right-radius: 0px solid white;
border-bottom-right-radius: 0px ;
border-bottom-left-radius: 3px;

}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>26</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel { 
background: white;
border: 1px solid #999999; 
padding: 2px 2px 2px 2px;

height: 30px;
width:26px;

border-top-left-radius: 0px ;
border-top-right-radius: 3px;
border-bottom-right-radius: 3px;
border-bottom-left-radius: 0px;

border-left: 0px solid white;
}</string>
         </property>
         <property name="pixmap">
          <pixmap>:/newPrefix/find.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel{
color:rgb(85, 85, 85);
font:bold;
font-family: Ubuntu, Helvetica, sans-serif;
}</string>
       </property>
       <property name="text">
        <string>Filtros</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbFilters">
       <property name="styleSheet">
        <string notr="true">QComboBox {
border: 1px solid #999999; 
border-radius: 4px;
padding: 2px 2px 2px 2px;
min-width: 6em;
}
QComboBox::down-arrow {
image: url(:newPrefix/arrow_down.png);
height: 10px;
}
QComboBox:editable {
background: white;
}
QComboBox::drop-down { border-style: none}
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="twItems">
     <property name="styleSheet">
      <string notr="true">QHeaderView::section {
border-radius: 3px;
padding: 2px 4px 2px 4px;

/* Fuente */
color: #3A3939;
font: bold 11px;
text-align: left;
font-family: Ubuntu, Helvetica, sans-serif;

background-color:rgb(244, 244, 244);
/*qlineargradient(spread:reflect, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(240, 240, 240, 255), stop:0.509174 rgba(250, 250, 250, 255), stop:0.963303 rgba(255, 255, 255, 255));*/
/* Bordes */
border-bottom: 1px solid #E3E3E3;
border-top: 1px solid #FFFFFF;
padding:6px;
border-top-left-radius: 0px;

}

QTableWidget{
font-family: Ubuntu, Helvetica, sans-serif;
font-size: 12px;
text-align: left;

border: 1px solid #BFBFBF;

}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::NoPen</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>24</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images_rc/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btSalir</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>776</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>796</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
